test/acceptance/test/add_tests.go

```diff
@@ -625,18 +636,20 @@ var _ = Describe("Weave GitOps Add Tests", func() {
 
 		By("Then I should see should see my workload deployed to the cluster", func() {
 			verifyWegoAddCommand(appName, WEGO_DEFAULT_NAMESPACE)
-			verifyWorkloadIsDeployed("nginx", "my-nginx")
+			verifyWorkloadIsDeployed(workloadName, workloadNamespace)
 		})
 	})
 
 	It("Verify that wego can deploy multiple workloads from a single app repo", func() {
 		var repoAbsolutePath string
-		appManifestFilePath1 := "./data/nginx.yaml"
-		appManifestFilePath2 := "./data/nginx2.yaml"
-		workloadName1 := "nginx"
-		workloadName2 := "nginx2"
-		workloadNamespace1 := "my-nginx"
-		workloadNamespace2 := "my-nginx2"
+		uniqueSuffix := RandString(6)
```

Comment:
How often is this block repeated? Can it be moved to a helper function?

Reply:
Yes good point, let me land this PR and I'll create a PR to move all the repeated logic into a helper function.